### DISCUSSION TITLE
help please

### DISCUSSION THREAD

**Discussion by sgiwaCW** [strimzi/strimzi-kafka-operator#7850] [Ideas]
please I need help, I tried to deploy kafka exporter using this yaml file
# Source: prometheus-kafka-exporter/templates/service.yaml
apiVersion: v1
kind: Service
metadata:
  name: release-name-prometheus-kafka-exporter
  annotations:
    {}
  labels:
    app: prometheus-kafka-exporter
    chart: prometheus-kafka-exporter-0.1.3
    release: release-name
    heritage: Helm
spec:
  type: ClusterIP
  ports:
    - port: 9308
      targetPort: exporter-port
      protocol: TCP
      name: http
  selector:
    app: prometheus-kafka-exporter
    release: release-name
---
# Source: prometheus-kafka-exporter/templates/deployment.yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  name: release-name-prometheus-kafka-exporter
  labels:
    app: prometheus-kafka-exporter
    chart: prometheus-kafka-exporter-0.1.3
    release: release-name
    heritage: Helm
spec:
  replicas: 1
  selector:
    matchLabels:
      app: prometheus-kafka-exporter
      release: release-name
  template:
    metadata:
      annotations:
        null
      labels:
        app: prometheus-kafka-exporter
        release: release-name
    spec:
      serviceAccountName: release-name-prometheus-kafka-exporter
      containers:
        - args:
            - '--kafka.server=kafka-server:9092'
          name: prometheus-kafka-exporter
          image: "danielqsj/kafka-exporter:latest"
          imagePullPolicy: IfNotPresent
          ports:
            - name: exporter-port
              containerPort: 9308
              protocol: TCP
          livenessProbe:
            httpGet:
              path: /
              port: exporter-port
          readinessProbe:
            httpGet:
              path: /
              port: exporter-port
          resources:
            {}

but I keep getting this error
   96375 memcache.go:255] couldn't get resource list for external.metrics.k8s.io/v1beta1: Got empty response for: external.metrics.k8s.io/

**Comment by scholzj** [comment]
How does this differ from the other thread? Also, this does not seem to have anything to do with Strimzi. That is not any kind of Strimzi deployment.

**Reply by sgiwaCW** [reply]
Thanks Scholzj.
I will prefer to use Strmzi. I have watch some videos from Strimzi>
Can you share with me if you have any helm chart for kafka exporter.
Thanks

**Reply by scholzj** [reply]
No. Strimzi is an operator which deploys and manages Apache Kafka. Not Kafka Exporter. Kafka Exporter is just a tool that can be deployed as part of our Kafka cluster in the `Kafka` custom resource.
